schema/postgresql/get_sla_ok_percent.sql

```diff
@@ -1,210 +1,173 @@
 CREATE OR REPLACE FUNCTION idoreports_get_sla_ok_percent(
-    id bigint,
-    starttime timestamp with time zone,
-    endtime timestamp with time zone,
-    sla_id integer DEFAULT NULL
-)
-    RETURNS float
-    LANGUAGE SQL
+	id INTEGER,
```

Comment:
why the change from `bigint` to `int` for `id`? AFAICT `object_id` is still `bigint` or `bigserial`, so this would require casting.